curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+Which attribute specifies a fallback for audio or video elements?
 
 #### --distractors--
 
-Placeholder distractor 1
+src
 
 ---
 
-Placeholder distractor 2
+controls
 
 ---
 
-Placeholder distractor 3
+alt
 
 #### --answer--
 
-Placeholder answer
+poster
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the ideal contrast ratio for large text?
 
 #### --distractors--
 
-Placeholder distractor 1
+2:1
 
 ---
 
-Placeholder distractor 2
+4.5:2
 
 ---
 
-Placeholder distractor 3
+3.5:1
 
 #### --answer--
 
-Placeholder answer
+3:1
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which online tool checks color contrast accessibility?
 
 #### --distractors--
 
-Placeholder distractor 1
+ColorZilla
 
 ---
 
-Placeholder distractor 2
+Canva
 
 ---
 
-Placeholder distractor 3
+Figma
 
 #### --answer--
 
-Placeholder answer
+Contrast Checker
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the purpose of the alt attribute in images?
 
 #### --distractors--
 
-Placeholder distractor 1
+To change image size
 
 ---
 
-Placeholder distractor 2
+To add a title
 
 ---
 
-Placeholder distractor 3
+To change image color
 
 #### --answer--
 
-Placeholder answer
+To provide a text alternative
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is a screen reader?
 
 #### --distractors--
 
-Placeholder distractor 1
+A browser for disabled users
 
 ---
 
-Placeholder distractor 2
+A CSS tool
+
 
 ---
 
-Placeholder distractor 3
+A web development library
 
 #### --answer--
 
-Placeholder answer
+A text-to-speech software
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the 'Inspect Element' feature do?
 
 #### --distractors--
 
-Placeholder distractor 1
+Edits text directly
 
 ---
 
-Placeholder distractor 2
+Downloads files
 
 ---
 
-Placeholder distractor 3
+Modifies images
 
 #### --answer--
 
-Placeholder answer
+Displays CSS and HTML
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which CSS property controls text size?
 
 #### --distractors--
 
-Placeholder distractor 1
+text-size
 
 ---
 
-Placeholder distractor 2
+size-font
 
 ---
 
-Placeholder distractor 3
+text-style
 
 #### --answer--
 
-Placeholder answer
+font-size
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the opacity property do in CSS?
 
 #### --distractors--
 
-Placeholder distractor 1
+Changes text size
 
 ---
 
-Placeholder distractor 2
+Adds a border
 
 ---
 
-Placeholder distractor 3
+Sets the element's position
 
 #### --answer--
 
-Placeholder answer
+Controls element transparency
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which browser is known for robust developer tools?
 
 #### --distractors--
 
-Placeholder distractor 1
+Safari
 
 ---
 
-Placeholder distractor 2
+Internet Explorer
 
 ---
 
-Placeholder distractor 3
+Opera
 
 #### --answer--
 
-Placeholder answer
+Firefox
```

Comment:
What exactly excludes Google Chrome from this list? While I personally dislike the browser, it does offer Google Lighthouse.

Reply:
Ya, going to agree here. This is very subjective and might not be a good question.

Reply:
You have not yet addressed this @kalvakeerthi